doc/README.hive

```diff
@@ -0,0 +1,153 @@
+Instructions For Running Hive
+------------------------------
+
+0) If necessary, download your favorite version of Hive off of Apache
+   and install it into a location where it's accessible on all cluster
+   nodes.  Usually this is on a NFS home directory.  You'll also need to
+   download a binary version of PostgreSQL
+
+   See 'Convenience Scripts' in README about
+   misc/magpie-apache-download-and-setup.sh, which may make the
+   downloading and patching easier.
+
+1) Select an appropriate submission script for running your job.  You
+   can find them in the directory submission-scripts/, with Slurm
+   Sbatch scripts using srun in script-sbatch-srun, Moab Msub+Slurm
+   scripts using srun in script-msub-slurm-srun, Moab Msub+Torque
+   scripts using pdsh in script-msub-torque-pdsh, and LSF scripts
+   using mpirun in script-lsf-mpirun.
+
+   You'll likely want to start with the base hive w/ hdfs script
+   (e.g. magpie.sbatch-srun-hive-with-hdfs) for your
+   scheduler/resource manager.  If you wish to configure more, you can
+   choose to start with the base script (e.g. magpie.sbatch-srun)
+   which contains all configuration options.
+
+2) Setup your job essentials at the top of the submission script.  As
+   an example, the following are the essentials for running with Moab.
+
+   #MSUB -l nodes : Set how many nodes you want in your job
+
+   #MSUB -l walltime : Set the time for this job to run
+
+   #MSUB -l partition : Set the job partition
+
+   #MSUB -q <my batch queue> : Set to batch queue
+
+   MOAB_JOBNAME : Set your job name.
+
+   MAGPIE_SCRIPTS_HOME : Set where your scripts are
+
+   MAGPIE_LOCAL_DIR : For scratch space files
+
+   MAGPIE_JOB_TYPE : This should be set to 'hive'
+
+   JAVA_HOME : B/c you need to ...
+
+3) Setup the essentials for Hive.
+
+   HIVE_SETUP : Set to yes
+
+   HIVE_VERSION : Set appropriately.
+
+   HIVE_HOME : Where your Hive code is.  Typically in an NFS
+   mount.
+
+   HIVE_LOCAL_DIR : A small place for conf files and log files local
+   to each node.  Typically /tmp directory.
+
+   HIVE_METASTORE_DIR : Path for Hive to use for the Hive Metastore
+
+   HIVE_DEFAULT_METASTORE_DIR : Default path if HIVE_METASTORE_DIR doesn't exist
+
```

Comment:
it's not clear to me why you have a HIVE_METASTORE_DIR and a HIVE_DEFAULT_METASTORE_DIR.  If HIVE_METASTORE_DIR doesn't exist, magpie-check-inputs should just output an error??

Reply:
At some point while I was integrating this into Magpie for some reason I thought we needed to have two different locations.  Definitely not the case, so HIVE_DEFAULT_METASTORE_DIR has been removed and now 'spark.sql.warehouse.dir' and 'hive.metastore.warehouse.dir' are set the same